src/app/components_ngrx/iteration-list-entry/iteration-list-entry.component.less

```diff
@@ -7,7 +8,9 @@
     padding-right: 10px;
     border-left: 4px solid;
     border-left-color: transparent;
-    .badge { background-color: @color-pf-black; }
+    .badge {
```

Comment:
Can you please remove this unused. We are not using this ```.badge``` class in iteration-list-entry component.